general/odoo_basics/add_user.rst

```diff
@@ -1,46 +1,141 @@
-=================
-How to add a user
-=================
+==================================
+Add Users and Manage Access Rights
+==================================
 
-Odoo provides you with the option to add additional users at any given
-point.
+Odoo defines a *user* as someone who has access to a database to perform daily tasks. You can add as
+many users as you need, and in order to restrict the type of information each user can access, rules
+can be applied. Users and access rights can be added and changed at any point.
```

Comment:
```suggestion
Odoo defines a *user* as someone who has access to a database to perform daily tasks. You can add as
many users as you need and, in order to restrict the type of information each user can access, rules
can be applied. Users and access rights can be added and changed at any point.
```

Reply:
I will keep the comma. When reading it out loud a break is taken + it is a sentence that, if removed, does not impact the understanding of the whole.

Reply:
If you keep the comma where it is, it means that you can write "You can add as many users as you need, rules can be applied." and I think it's not right.
What's right would be: "You can add as many users as you need and rules can be applied."

If so, then the comma must move where I suggest :)

Reply:
Agree. It slipped out.
I misread the first suggestion. Though it was just suggested to remove it. Sorry! I will be more careful the next time.
Thanks!